### DISCUSSION TITLE
Sorting in main document view

### DISCUSSION THREAD

**Discussion by dermb** [LeoKlaus/Paperparrot#113] [Feature Requests]
As a document library maintainer, I would like to sort the main document view (without search operation) similar to paperless-ngx web view,  mainly by date added, to quickly access recently added documents.

**Comment by dermb** [comment]
A small addendum: Of course it’s possible to define a custom sorted view in the paperless-ngx web interface and use it as default view in Paperparrot which achieves the purpose completely. But not in Paperparrot on its own.

**Comment by LeoKlaus** [comment]
This has been on my todo list for quite some time, I'll see to implement it with the next release!
